assets/scss/modules/_buttons.scss

```diff
@@ -50,7 +50,26 @@
           border: 1px solid $dp-color-darkgreen;
         }
       }
+
+      &grey {
+        background-color: $dp-color-grey;
+        border: 1px solid $dp-color-grey;
+        color: $dp-color-white;
+
+        &:hover {
+          background-color: $dp-color-darkgrey;
+          border: 1px solid $dp-color-darkgrey;
+        }
+      }
     }
   }
+
+  .dp-button:disabled {
```

Comment:
Only to learn, how do you guarantee that these styles have priority over color styles?

Reply:
in this case, it works because of the cascade effect and not for the CSS specificity due to the class color and the disabled class color have the same weight (specificity)